tox.ini

```diff
@@ -19,12 +18,14 @@ setenv =
 deps =
     -rrequirements.txt
     -rtest-requirements.txt
-    ansible24: ansible==2.4.6.0
-    ansible25: ansible==2.5.9
-    ansible26: ansible==2.6.4
+    ansible24: ansible>=2.4,<2.5
```

Comment:
I think, author just didn't think about it. I stole this technique from ansible-lint and I think it's better this way.
Just think about it: within 2.x all "patch" subversions should  still have the same public API.
So we are testing just compatibility with those API, not the full matrix of all possible combos, because we don't need to test Ansible Engine from within Molecule tests, we test Molecule itself after all...